staging/script.xsqueeze/resources/lib/classes/NowPlayingWindow.py

```diff
@@ -399,6 +440,19 @@ def onClick( self, control ):
             log("Button event: EXIT")
             self.exitXSqueeze()
 
+         #hdmi support
+          if (control == constants.BUTTONVOLUP) and constants.CECSUPPORT == 'true':
+            cec.volume_up()
```

Comment:
A loop would be better. I will change it in my next commit. I will also link this loop with the volume factor (see settings.xml). The number of calls will depend on the volume factor then.